### ISSUE TITLE
Why does flickity set tabindex="0" on the container?

### ISSUE THREAD

**Issue by baloneysandwiches** [metafizzy/flickity#163]
While testing navigability from the keyboard using tab and shift-tab, I had a mysterious blank slot when visual focus indicators went missing, and traced it back to the Flickity container. This seems weird, especially because space-bar-ing and enter-ing while focused on the container don't have any default effect on the slideshow. It seems to me that the default behavior should be no `tabindex`.

Accessibility folks generally seem to agree that if you're using `tabindex` at all, it's typically a sign that something's wrong your your markup. All's good with the Flickity `<button>` markup, so what's does adding `tabindex` on init do for the container? Could there be a declarative option to turn this off?

P.S. Thanks for another epic layout plugin. Huge props.

**Comment by desandro** [comment]
Hey! Thanks for this legit inquiry.

`tabindex=0` allows the Flickity element to be tab-able. You can focus it via keyboard. Then, you can press ← or → on the keyboard to increment through the carousel. [Using `tabindex=0` for carousels is recommended by W3C](http://www.w3.org/WAI/tutorials/carousels/functionality/#focusing-carousel-items)

As for actions after hitting Space or Enter, I haven't added functionality for these keys yet. I've left that open in case a developer might make use of them.

If you have suggestions for improving accessibility with Flickity, I'd love to hear them. I do need help determining the best way to implement ARIA, see #66. 

> Could there be a declarative option to turn this off?

Set [`accessibility: false`](http://flickity.metafizzy.co/options.html#accessibility)

> I had a mysterious blank slot when visual focus indicators went missing, and traced it back to the Flickity container.

Flickity currently removes focus style. I realize this is controversial, but I have [documented how to re-enable it](http://flickity.metafizzy.co/style.html#gallery).

**Comment by baloneysandwiches** [comment]
Thank you for such a prompt and thoughtful reply.

> Using tabindex=0 for carousels is recommended by W3C

Okay, great, but where on that page does it say that? When I view source on the "previous and next buttons" demo on that page, there is no `tabindex` set on the carousel.

The arrow key binding is cool. I hadn't figured that out. It totally makes sense for an ultra minimal UI where there are no previous/next buttons. I guess I'm just wishing that when prev/next buttons option is `true`, there is no container `tabindex`. Thoughts?

**Comment by desandro** [comment]
You can still tab to left & and right buttons. I prefer to maintain current functionality.
